packages/react/src/components/TextField/index.tsx

```diff
@@ -13,7 +13,7 @@ export interface TextFieldProps
     value: string,
     e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => void;
-  fieldRef: React.Ref<HTMLInputElement | HTMLTextAreaElement>;
+  fieldRef?: React.Ref<HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
This should already be optional as the default value is set below, but this is just updating the type to match the expected interface.